packages/react-native-bpk-component-button/src/BpkBorderedButton.android.js

```diff
@@ -58,6 +55,7 @@ const styles = StyleSheet.create({
 export type Props = {
   backgroundColor: string,
   borderColor: string,
+  borderRadius: number,
```

Comment:
Could this also be a string based on the types 'sm', 'lg', 'pill'? Therefore flow would give an error?

Reply:
`BpkTouchableOverlay` accepts a string or number, but `BpkButton` only ever uses numbers for it so it's still valid.